sklearn/ensemble/_stacking.py

```diff
@@ -39,22 +39,25 @@ class _BaseStacking(TransformerMixin, _BaseHeterogeneousEnsemble,
 
     @abstractmethod
     def __init__(self, estimators, final_estimator=None, cv=None,
-                 stack_method='auto', n_jobs=None, verbose=0):
+                 stack_method='auto', n_jobs=None, verbose=0,
+                 pass_through=False):
```

Comment:
When we elsewhere use "passthrough" I don't think pass_through is a good idea

Reply:
Ah good catch, I'll update to be consistent with `passthrough`.